apps/costcontrol/js/app.js

```diff
@@ -505,6 +507,10 @@ var CostControlApp = (function() {
     showAppDetailView: function _showAppDetailView() {
       window.location.hash = '##appdetail-view';
     },
-    _onHashChange: _onHashChange
+    _onHashChange: _onHashChange,
+    _startApp: startApp,
```

Comment:
We do we need to expose `startApp()` method? You can force a call to `startApp` by [sending a `message`](https://github.com/gitmai/gaia/blob/bug-1115509-lets-go-button-does-not-response/apps/costcontrol/js/app.js#L416) of type `fte_finished`.